### ISSUE TITLE
New device can’t add

### ISSUE THREAD

**Issue by phy0930** [rospogrigio/localtuya#1307]
## The problem
I have a energy recover ventilation device. It's control panel used tuya chips(show tuya iot name is "xfj"). Local tuya support device list doesn't show it.
I used Localtuya integration and key-in correct IP, ID and LocalKey. But it shows "Connection to device succeeded but no datapoints found, please try again. Create a new issue and include debug logs if problem persists."
It doesn't show any message about localtuya issue from Home Assistant log after I add new device.

## Environment
- Localtuya version: v5.0.0 
- Home Assistant Core version: 2023.2.4 
- [] Does the device work using the Home Assistant Tuya Cloud component ? 
- [] Does the device work using the Tinytuya (https://github.com/jasonacox/tinytuya) command line tool ? 
- [] Was the device working with earlier versions of localtuya ? Which one? 
- [x] Are you using the Tuya/SmartLife App in parallel ?

Thank you very much.
